sirepo/server.py

```diff
@@ -519,6 +521,11 @@ def api_staticFile(path_info=None):
     Returns:
         flask.Response: flask.send_from_directory response
     """
+    if re.match(r'en/.*html', path_info):
```

Comment:
`path_info` can be None, so protect against this. 

I changed my mind. The analytics code is copied N times so what we want to do is something like this:

```py
if _google_tag_manager and re.match(r'en/.*html', path_info or ''):
    return _google_tag_manager_re.sub(
        _google_tag_manager, 
        pkio.read_text(simulation_db.STATIC_FOLDER.join(path_info)),
    )
```

At the top of the file:

```py
#: If google_tag_manager_id set, string to insert in landing pages for google analytics
_google_tag_manager = None

#: what to match in landing pages to insert `_google_tag_manager`
_google_tag_manager_re = re.compile('(?=</head>)', flags=re.IGNORECASE)
```

Then add to`init()`:

```py
    if cfg.google_tag_manager_id:
        _google_tag_manager = f'''<script>
(function(w,d,s,l,i){{w[l]=w[l]||[];w[l].push({{'gtm.start':new Date().getTime(),event:'gtm.js'}});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);}})(window,document,'script','dataLayer','{cfg.google_tag_manager_id}');
</script>'''
```